{{cookiecutter.project_slug}}/.github/workflows/tests.yml

```diff
@@ -38,4 +45,4 @@ jobs:
           hatch run lint:all
       - name: Run Tests
         run: |
-          hatch run test:pytest
+          hatch test
```

Comment:
Nice, the new way of doing this! Thanks.

Reply:
My idea was to leave the testing for different versions and environments mainly with GitHub. This is achieved by setting the matrix for os /  python version in the GitHub action and executing a hatch test in the standard hatch-test environment.
I was only struggling a little bit with the package dependencies, there might be some room for optimization.